Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM nginx:alpine
```

Comment:
use always a tagged version of the image, i.e. nginx:1.22 etc. Otherwise in 2 years nginx:alpine will have nginx 2.0 and your config may break resulting in downtime